src/ip/QosConfig.h

```diff
@@ -43,12 +43,13 @@ class acl_nfmark
     CBDATA_CLASS(acl_nfmark);
 
 public:
-    acl_nfmark() : next(NULL), aclList(NULL), nfmark(0) {}
+    acl_nfmark() : next(NULL), aclList(NULL), nfmark(0), nfmask(0xffffffff) {}
     ~acl_nfmark();
 
     acl_nfmark *next;
     ACLList *aclList;
     nfmark_t nfmark;
+    nfmark_t nfmask;
```

Comment:
Please use a new `ConnMarkConfig nfMark` field instead of these two low-level fields.